js/classes.js

```diff
@@ -0,0 +1,43 @@
+import { teachers } from "./teachers.js";
+const classes = [
+    {
+      title: "Classes",
+      details: [
+        { class: "Fullstack" },
+        { class: "Cloud" },
+        { class: "AWS" },
+        { class: "Javascript" },
+        { class: "Java" },
+      ],
+    },
+  ];
+
+  function renderClasses() {
+    const dynamicContent = document.getElementById("dynamic-content");
+    dynamicContent.innerHTML = "";
+
+    // Classes başlığını ekleyin
+    const classesHeader = `<h4 id="classes-header">Classes</h4>`;
+    dynamicContent.innerHTML = classesHeader;
+
+    classes[0].details.slice(0, 2).forEach((pClass) => {
```

Comment:
You processing data and using html templates in one file. This makes it hard to read and change later on.
Sepaarate data handling and dom handling from each other